[2025-08-28T08:25:36.608+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.extract manual__2025-08-28T08:25:34.734597+00:00 [queued]>
[2025-08-28T08:25:36.650+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.extract manual__2025-08-28T08:25:34.734597+00:00 [queued]>
[2025-08-28T08:25:36.650+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-08-28T08:25:36.679+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract> on 2025-08-28 08:25:34.734597+00:00
[2025-08-28T08:25:36.704+0000] {standard_task_runner.py:60} INFO - Started process 568 to run task
[2025-08-28T08:25:36.745+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'extract', 'manual__2025-08-28T08:25:34.734597+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp2xim9qo4']
[2025-08-28T08:25:36.801+0000] {standard_task_runner.py:88} INFO - Job 8: Subtask extract
[2025-08-28T08:25:36.994+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-28T08:25:37.146+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline.extract manual__2025-08-28T08:25:34.734597+00:00 [running]> on host dd18df0f367e
[2025-08-28T08:25:37.345+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-08-28T08:25:34.734597+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-28T08:25:34.734597+00:00'
[2025-08-28T08:25:37.349+0000] {logging_mixin.py:188} INFO - Extracting (fake)...
[2025-08-28T08:25:39.361+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-08-28T08:25:39.442+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=extract, execution_date=20250828T082534, start_date=20250828T082536, end_date=20250828T082539
[2025-08-28T08:25:39.541+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-28T08:25:39.685+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
